tests/test_contrib.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow contrib."""
+
+import pytest
+from marshmallow import Schema, ValidationError
+
+from marshmallow_utils.fields import contrib
```

Comment:
I think what Guillaume meant is that you can import directly (since you added them to the `__init__`)
```suggestion
from marshmallow_utils.fields import Function, Method
```